src/notifications/ManagerLinux.cpp

```diff
@@ -44,29 +52,17 @@ NotificationsManager::postNotification(const QString &roomid,
                                        const QString &text,
                                        const QImage &icon)
 {
-        uint id             = showNotification(roomname, sender + ": " + text, icon);
-        notificationIds[id] = roomEventId{roomid, eventid};
-}
-/**
- * This function is based on code from
- * https://github.com/rohieb/StratumsphereTrayIcon
- * Copyright (C) 2012 Roland Hieber <rohieb@rohieb.name>
- * Licensed under the GNU General Public License, version 3
- */
-uint
-NotificationsManager::showNotification(const QString summary,
-                                       const QString text,
-                                       const QImage image)
-{
+        Q_UNUSED(icon)
+
         QVariantMap hints;
-        hints["image-data"] = image;
+        hints["image-data"] = sender + ": " + text;
         hints["sound-name"] = "message-new-instant";
         QList<QVariant> argumentList;
-        argumentList << "nheko"; // app_name
-        argumentList << (uint)0; // replace_id
-        argumentList << "";      // app_icon
-        argumentList << summary; // summary
-        argumentList << text;    // body
+        argumentList << "nheko";  // app_name
+        argumentList << (uint)0;  // replace_id
+        argumentList << "";       // app_icon
+        argumentList << roomname; // summary
+        argumentList << text;     // body
```

Comment:
This was sender + ": " + text before, right?

Reply:
No, `roomname` was passed to `showNotification` as `summary`